azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/Pump.java

```diff
@@ -12,18 +12,25 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 
-class Pump {
+class Pump extends Closable {
```

Comment:
renaming this class to `PumpManager` or something equivalent could help with readability.